.gitignore

```diff
@@ -5,3 +5,9 @@ dist
 node_modules
 package
 **/bundle.js
+
+# Editor settings: Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm
+*.iml
+.idea/
+*.ipr
+*.iws
```

Comment:
Ideally these should go in your gloabl git ignore file, so that we do not have to add them to every project.

Reply:
done.